Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeDrawingAreaProxyIOS.mm

```diff
@@ -99,14 +99,14 @@ - (void)setPreferredFramesPerSecond:(NSInteger)preferredFramesPerSecond
 - (void)displayLinkFired:(CADisplayLink *)sender
 {
     ASSERT(isUIThread());
-    _drawingAreaProxy->didRefreshDisplay();
+    _drawingAreaProxy->didRefreshDisplay((WebKit::WebProcessProxy*)nullptr);
```

Comment:
This is a little too explicit.  Just nullptr ought to do it.

Reply:
It was needed since we had two overloads of didRefreshDisplay that took a pointer (even though one was private, but ambiguous overloads fail before access specifiers are evaluated).

Changed the signatures a bit to avoid this.